students/jacob_daylong/Lesson 3/MailRoom/mailroom.py

```diff
@@ -0,0 +1,76 @@
+﻿
+#Write a small command-line script called mailroom.py. This script should be executable. The script should accomplish the following goals:
+
+#It should have a data structure that holds a list of your donors and a history of the amounts they have donated.
+#This structure should be populated at first with at least five donors, with between 1 and 3 donations each. You can store that data structure in the global namespace.
+#The script should prompt the user (you) to choose from a menu of 3 actions: “Send a Thank You”, “Create a Report” or “quit”.
+
+TotalGiven = 0.00
```

Comment:
All of these variables should be snake_case: total_given, full_name, num_gifts, etc.